src/basic_recipes/poly.jl

```diff
@@ -62,8 +64,10 @@ poly_convert(mesh::GeometryBasics.Mesh, transform_func=identity) = mesh
 
 function poly_convert(polygon::Polygon, transform_func=identity)
     outer = metafree(coordinates(polygon.exterior))
-    points = Vector{Point2f}[apply_transform(transform_func, outer)]
-    points_flat = Point2f[outer;]
+    # TODO consider applying f32 convert here too. We would need to identify this though...
```

Comment:
How would that work if the Float32 convert changes though?

Reply:
Add `f32_conversion_obs` here
https://github.com/MakieOrg/Makie.jl/blob/14b65a0e8c786030f095cfe37860e6eaeec39eaa/src/basic_recipes/poly.jl#L121
pass it through and apply it, and add a "post f32 convert" space (world32?) for backends to recognize.